gadgets/trace_exec/gadget.yaml

```
@@ -79,6 +79,12 @@ structs:
         width: 5
         alignment: left
         ellipsis: end
+    - name: pupper_layer
+      description: Whether the executable's parent is in the upper layer of the overlay filesystem
```

Suggestion:
```suggestion
      description: Whether the executable's parent in the process hierarchy is in the upper layer of the overlay filesystem. Only initialized when the execution succeeded.
```
Ditto for upper_layer
Ditto in the documentation.